server/settings/publishers.py

```diff
@@ -1505,6 +1511,11 @@ class PublishersModel(BaseSettingsModel):
         "optional": False,
         "active": True
     },
+    "ValidateAnimationProductTypePublish": {
```

Comment:
Ignore me - only **now** I realize this validates whether it publishes at least ONE product type instead of none.... but... it ignores the export of USD - which I believe is also supported, no?

Reply:
yes, I think we should also support the usd too

Reply:
@BigRoy do you have any clue on the multiverse usd animation extractor? Should we include the check on this or no?

Reply:
No idea - I haven't used that one.. but I suppose we may need to, but isn't that a product type of its own or something? (I vaguely recall)